src/sentry/ingest/consumer/processors.py

```diff
@@ -186,6 +191,7 @@ def process_event(
                 start_time=start_time,
                 event_id=event_id,
                 project_id=project_id,
+                queue=split_queue_router.route_for_queue("events.save_event_transaction"),
```

Comment:
You do not need this. 
The way to use the router in this scenario is to configure Celery with the router 

```
CELERY_ROUTES = ("sentry.queue.routers.SplitQueueTaskRouter",)
```

Then set up `CELERY_SPLIT_QUEUE_TASK_ROUTES `for the task you are interested into  and remove the queue from the definition of `save_event_transaction`. This step will make the router return the default queue.

Then restart the workers to consume the split queues on top of the default one.

Then roll out your queue by changing the `celery_split_queue_task_rollout` options